### ISSUE TITLE
option --ban-relative-imports: 'const' must not be supplied for action 'store'

### ISSUE THREAD

**Issue by ikonst** [adamchainz/flake8-tidy-imports#306]
### Python Version

3.7.9

### Package Version

4.4.0

### Description

flake8 fails with:
```
  File "lib/python3.7/site-packages/flake8_tidy_imports/__init__.py", line 48, in add_options
    help="Ban relative imports, from parental modules or in all cases.",
...
optparse.OptionError: option --ban-relative-imports: 'const' must not be supplied for action 'store'
```

**Comment by adamchainz** [comment]
What flake8 version are you using? And can you give the full Python version number?

**Comment by ikonst** [comment]
This seems to be latest cpython behavior:
https://github.com/python/cpython/blob/64a7812c170f5d46ef16a1517afddc7cd92c5240/Lib/optparse.py#L690-L694

**Comment by adamchainz** [comment]
Okay so the problem is that using const like this is an argparse feature, and flake8 only switched from optparse to argparse in version 3.8.0. I've released version 4.4.1 with the correct dependency info, and yanked version 4.4.0.

I'd recommend you upgrade flake8, or downgrade flake8-tidy-imports for the time being.
